prometheus/base/alerting/prometheus.pod_alerts.yaml

```diff
@@ -35,3 +35,11 @@ spec:
       annotations:
         summary: "Pod {{ $labels.pod }} is not ready"
         description: "Pod {{ $labels.pod }} in namespace {{ $labels.namespace }} is not ready for more than 15 minutes."
+    - alert: PersistentVolumeIssues
+      expr: kube_persistentvolume_status_phase{phase=~"Failed|Pending", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"} == 1
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        summary: "Pod {{ $labels.pod }} is not having sufficient Persistent volume"
+        description: "Pod {{ $labels.pod }} in namespace {{ $labels.namespace }} is not having enough Persistent volume"
```

Comment:
1. we need the pv name on the message as well.
2. I don't know if this statement is necessarily correct. it might be that the issue is different. I'd rephrase it saying that the pv is in the {{ $labels.phase }} for more than 5 minutes.

Reply:
I agree, when running the query `kube_persistentvolume_status_phase`, pod label is not a part of the labels that we get, so it is not usable here.
The query is about the PV, not the pod and it should be included in the message.
So maybe something like:  
"Persistent Volume {{ $labels.persistentvolume }} in namespace {{ $labels.namespace }} is in {{ $labels.phase }} phase for more than 5 minutes"